src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_io_net_stm32fx_nucleo_wifi.c

```diff
@@ -50,12 +51,39 @@ xi_bsp_io_net_state_t xi_bsp_io_net_create_socket( xi_bsp_socket_t* xi_socket )
     return XI_BSP_IO_NET_STATE_OK;
 }
 
+static xi_bsp_io_net_state_t xi_bsp_io_net_configure_tls( const char* host )
+{
+    WiFi_Status_t status          = WiFi_MODULE_SUCCESS;
+    posix_time_t current_datetime = xi_bsp_time_sntp_getseconds_posix();
+    uint8_t* tls_mode             = ( uint8_t* )"o"; //["m"utual || "o"ne-way]
+    char* tls_cert                = GLOBALSIGN_ROOT_CERT;
+
```

Comment:
does this mean that we have to add logic to attempt connections with different certificates if the first attempt fails?  I think Bob did something similar for a device at one point but I don't know if we've ever really thought through what this really should look like.